elements/pl-python-variable/pl-python-variable.css

```diff
@@ -2,7 +2,7 @@
     display: block;
     overflow-x: scroll;
     font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
-    font-size: 12px;
+    font-size: 1rem;
```

Comment:
Could we just delete the `font-size` property entirely and use the Bootstrap defaults?

Reply:
Good thinking! @ffund does it look sensible if we just don't set a `font-size` at all? I'm also tempted to remove `font-family` unless there's a good reason we are specifying it?

Reply:
Sounds fine. if we do that, we should also remove `font-family` for the 'dimensions' line at the bottom.